crates/e2e/Cargo.toml

```diff
@@ -52,6 +52,10 @@ scale-info = { workspace = true, features = ["derive"] }
 [features]
 default = ["std"]
 std = []
+live-state-test = [
+    "subxt/unstable-light-client",
```

Comment:
What's the purpose of this, do we need to use `smoldot` to communicate with chopsticks? It looks like we just use the normal RPC?

Reply:
I assume yes, we need a light client to communicate with a node.

Reply:
The initialisation code uses a regular `RpcClient` https://github.com/paritytech/ink/pull/1949/files#diff-4f8810a386176bf775fb827b07ad2d012aef2f57638a97ffad1fc7d898e61395R125, so no usage of the light client rpc AFAIU.

Does it still work without this feature enabled?

Reply:
Yes, but I have very limited knowledge whether there is the edge case when it won't work

Reply:
> The initialisation code uses a regular `RpcClient` https://github.com/paritytech/ink/pull/1949/files#diff-4f8810a386176bf775fb827b07ad2d012aef2f57638a97ffad1fc7d898e61395R125, so no usage of the light client rpc AFAIU.
> 
> Does it still work without this feature enabled?

Apparently, it is possible to `LightClient` because it shares the same interface as `RpcClient` but I don't see the difference for local testing. Need to consult @cmichi